openedx/core/djangoapps/catalog/cache.py

```diff
@@ -2,4 +2,8 @@
 PROGRAM_CACHE_KEY_TPL = 'program-{uuid}'
 
 # Cache key used to locate an item containing a list of all program UUIDs.
+# This has to be deleted when removing the waffle flags populate-multitenant-programs and get-multitenant-programs
```

Comment:
Please include the associated ticket number in this comment (e.g., `For more, see LEARNER-1146`).